src/components/Toolbar/Toolbar.tsx

```diff
@@ -54,7 +54,37 @@ const Toolbar: FC<Props> = ({ onWidgetDrag }) => {
         onDragStart={dragStartHandler}
         onDragEnd={dragEndHandler}
       >
-        Text
+        <WidgetItem icon="text" text={t('widget_item.text')} />
+      </div>
+      <div
```

Comment:
can we extract this div
` <div
        draggable
        unselectable="on"
        data-widget-type="visualization"
        onDragStart={dragStartHandler}
        onDragEnd={dragEndHandler}
      >` to separate component to avoid repetitions?